Dockerfile

```diff
@@ -1,14 +1,15 @@
 # Building the main container
-FROM ubuntu:20.04
+FROM python:3.8-buster
```

Comment:
Python image based on debian, while we prefer ubuntu, because it updates much faster do we receive update for critical vulnerabilities in packages frequently.

Reply:
talking about the security of debian vs ubuntu is a discussion of its own imo. When looking at docker images I'm personally also concerned about speed, stability and weight of the image itself. To me it seems counter intuitive not to use the official python image for something based on python. But this is a valid [dicussion](https://pythonspeed.com/articles/base-image-python-docker-images/) and I'm no expert by any means on this topic.

Reply:
You can compare it by yourself https://docs.docker.com/engine/scan/:
```
docker scan python:3.8-buster
docker scan ubuntu:20.04
```

My results are:
```
python:3.8-buster -- Tested 431 dependencies for known vulnerabilities, found 381 vulnerabilities.
ubuntu:20.04 -- Tested 93 dependencies for known vulnerabilities, found 16 vulnerabilities.
```

Reply:
I do understand that your main priority seems to be the security aspect and it's perfectly fine to prefer ubuntu as a base image. I'd just like to add, that basing your decision on number of CVEs alone may may miss [this](https://github.com/docker-library/faq#why-does-my-security-scanner-show-that-an-image-has-cves) aspect.

Reply:
I've just showed you an example that is free to use, we have other "paid" security scanners which shows other results, but the overall comparison is the same.